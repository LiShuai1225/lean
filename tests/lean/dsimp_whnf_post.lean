structure Foo := (foo : unit)

def my_foo : Foo := Foo.mk ()
@[simp] lemma my_foo.def : my_foo = Foo.mk () := rfl

example : Foo.foo my_foo = () := by dsimp
