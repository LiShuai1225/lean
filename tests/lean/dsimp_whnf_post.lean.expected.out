dsimp_whnf_post.lean:6:36: error: tactic failed, there are unsolved goals
state:
‚ä¢ () = ()
